README.md

```diff
@@ -1,2 +1,30 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+<h3>(a)
+  
+The algorithm checks through the list to see if it has duplicate entries in it. If it has, it returns true, if not it returns false.
+<h3>(b)
+  
+The algorithm is in quadratic time (O(n^2)) because the function checks item i in the list, then j increments through all posible positions in the list to check for duplicates. This means that for each item in the list you are checking the entire list.
+<h3>(c) 
+  
+Because it is still checking every item in the list and seeing if it is duplicated, but is checking more efficiently.
+<h3>(d)
+  
+The algorithm runs faster because j is only iterating up to i - 1. Which means that at the start it is checking less items. 
```

Comment:
But why approximately twice as fast? The key is to notice the original algorithm checks each pair twice (i.e. (a,b) and (b,a)) whereas the new one checks each only once.